mlflow/ml-package-versions.yml

```diff
@@ -42,7 +42,9 @@ pytorch-lightning:
   autologging:
     minimum: "1.0.5"
     maximum: "1.5.10"
-    requirements: ["torchvision", "scikit-learn"]
+    requirements:
+      "< 1.2.0": ["torch<1.11.0", "torchvision", "scikit-learn"]
+      ">= 1.2.0": ["torchvision", "scikit-learn"]
```

Comment:
In the failed build `torch` 1.11.0 was used. This indicates `torch` 1.11.0 introduced a change that's incompatible with older versions of `pytorch-lightning`.

### Successful case

https://github.com/mlflow/mlflow/runs/5476435114?check_suite_focus=true#step:11:50

```
torch                     1.10.2      2022-01-27  
```

### Failed case

https://github.com/mlflow/mlflow/runs/5526315925?check_suite_focus=true#step:11:16

```
torch                     1.11.0      2022-03-10
```